Comparing source compatibility of /home/someone/git/com.github/io7m/smfj/com.io7m.smfj.format.obj/target/com.io7m.smfj.format.obj-0.11.0.jar against /home/someone/.m2/repository/com/io7m/smfj/com.io7m.smfj.format.obj/0.11.0-SNAPSHOT/com.io7m.smfj.format.obj-0.11.0-SNAPSHOT.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.smfj.format.obj.SMFFormatOBJ  (not serializable)
	+++  NEW METHOD: PUBLIC(+) java.lang.String toString()
***! MODIFIED CLASS: PUBLIC FINAL com.io7m.smfj.format.obj.SMFOBJImporter  (not serializable)
	+++  NEW METHOD: PUBLIC(+) void parse()
	---! REMOVED METHOD: PUBLIC(-) void parseData()
		---  REMOVED EXCEPTION: java.lang.IllegalStateException
	---! REMOVED METHOD: PUBLIC(-) void parseHeader()
	---! REMOVED METHOD: PUBLIC(-) boolean parserHasFailed()
